src/users/components/invite-user-permissions/invite-user-permission.component.html

```diff
@@ -1,36 +1,36 @@
   <!--permissions -->
   <ng-container [formGroup]="inviteUserForm">
     <lib-gov-uk-form-group-wrapper
-      [config]="{hint: 'Choose at least one action the user can do. You can change this later.', legend: 'Permissions', key: 'roles'}"
+      [config]="{hint: 'Choose at least one action the user can do. You can change this later.', legend: 'What user Permissions would you like to approve?', key: 'roles'}"
```

Comment:
Not sure if lint will pick this one up but it's a _really_ long line and not easy to read as it requires sideways scrolling...

Reply:
split the line